hidapi.podspec

```diff
@@ -0,0 +1,30 @@
+Pod::Spec.new do |spec|
+
+  spec.name         = "hidapi"
+  spec.version      = "0.9.1"
+  spec.summary      = "A Simple library for communicating with USB and Bluetooth HID devices on Linux, Mac and Windows."
+
+  spec.description  = <<-DESC
+  HIDAPI is a multi-platform library which allows an application to interface with USB and Bluetooth HID-Class devices on Windows, Linux, FreeBSD, and macOS. HIDAPI can be either built as a shared library (.so, .dll or .dylib) or can be embedded directly into a target application by adding a single source file (per platform) and a single header.
+                   DESC
+
+  spec.homepage     = "https://github.com/libusb/hidapi"
+
+  spec.license      = { :type=> "One of three licenses", :file => "LICENSE.txt" }
```

Comment:
Not sure how better describe type of 'select one of three available licenses'. Any suggestions?

Reply:
I'd say:
"GNU GPLv3 or BSD or HIDAPI original"